### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jsfr**
Updates all dependencies to their latest version.

### CODE REVIEW

**Review by MiguelMRoldan — APPROVED** [review]
Looks good to me

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node 21.2.0
```
Comment (r0binary) [inline .tool-versions:1]: Some quick googling revealed this file is used by asdf. Is that the case here too?I feel at this point asking might be faster than continued search

**Review by r0binary — APPROVED** [review]
Thanks @jsfr. But shouldn't this be covered by renovate?

**Comment by jsfr** [comment]
> Thanks @jsfr. But shouldn't this be covered by renovate?

Not e.g. the upgrade to Node 21 in our workflow files, I believe. We were in a weird state here because we use Node 21 in the Dockerfile but not anywhere else, meaning build, local and test ran with Node 20, but the production Dockerfile ran with Node 21.

Also Renovate is slow and I need a new release with everything updated for the task this PR is related to :)

File: .tool-versions (hunk 1 above)
Comment (jsfr) [reply .tool-versions:1]: `.tool-versions` can be used by many different things, but e.g. asdf yes. Committing this file means it's easier to ensure everyone uses the same version locally, it's not a hard requirement though.

**Pull request merged**